it/uk/gov/hmrc/agentaccesscontrol/support/BaseISpec.scala

```diff
@@ -113,19 +113,19 @@ trait DesStub[A] {
       this
     }
 
-    def andHasNoRelationInDesWith(client: SaUtr): A = {
+    def andHasNoRelationInDesWith(client: TaxIdentifier): A = {
       stubFor(matcherForClient(client).willReturn(aResponse().withStatus(404)))
       this
     }
 
-    def andIsRelatedToClientInDes(clientUtr: SaUtr, authorizationHeader: String = "secret", envHeader: String = "test"): DesStubBuilder = {
+    def andIsRelatedToClientInDes(clientUtr: TaxIdentifier, authorizationHeader: String = "secret", envHeader: String = "test"): DesStubBuilder = {
       new DesStubBuilder(clientUtr, authorizationHeader, envHeader)
     }
 
-    private def matcherForClient(client: SaUtr) =
+    private def matcherForClient(client: TaxIdentifier) =
```

Comment:
The client parameter has been changed to TaxIdentifierbut the path still has sa in it